bin/lib/notify.py

```diff
@@ -0,0 +1,134 @@
+import urllib.request
```

Comment:
For what it's worth I've found `PyGithub` a great library for building GH integrations in python. https://github.com/PyGithub/PyGithub

Not saying we have to use it (of course!!) but it might make things easier if we choose to extend `ce`'s GH abilities.

Also...it doesn't appear to support GraphQL anyway (I just checked!)